bedrock/mozorg/templates/mozorg/about/governance/policies/security/certs/policy.html

```diff
@@ -24,892 +24,990 @@
-(EKU)</a> extension specifying all extended key usages that the subordinate CA is
-  authorized to issue certificates for. The anyExtendedKeyUsage KeyPurposeId MUST
-  NOT appear within this extension.</p>
-<p>If the certificate includes the id-kp-serverAuth extended key usage, then to be considered
-  technically constrained, the certificate MUST be Name Constrained as described
-  in section 7.1.5 of version 1.3 or later of the <a href="https://cabforum.org/baseline-requirements-documents/">Baseline Requirements</a>.
+  <p>CA operators MUST maintain current best practices to prevent
+  algorithm attacks against certificates. As such, all new certificates
+  MUST have a serial number greater than zero, containing at least 64 bits of
+  output from a CSPRNG.</p>
+  <p>CA operators MUST NOT issue certificates, CRLs, or OCSP responses, that have:</p>
+  <ul class="mzp-u-list-styled">
+    <li>ASN.1 DER encoding errors;</li>
+    <li>invalid public keys (e.g., RSA certificates with public exponent
+        equal to 1); <em>or</em></li>
+    <li>missing or incorrect extensions (e.g., TLS certificates with no subjectAltName extension, delegated OCSP responders without the id-pkix-ocsp-nocheck extension, partial/scoped CRLs that lack a distributionPoint in a critical issuingDistributionPoint extension).</li>
+  </ul>
+  <p>CA operators MUST NOT issue certificates that have:</p>
+  <ul class="mzp-u-list-styled">
+    <li>duplicate issuer names and serial numbers (except that a Certificate
+        Transparency pre-certificate is allowed to match the corresponding
+        certificate); <em>or</em></li>
+    <li>cRLDistributionPoints or OCSP authorityInfoAccess extensions for
+        which no operational CRL or OCSP service exists.</li>
+  </ul>
+  <p>CA operators MUST NOT generate the key pairs for end entity certificates that have an
+  EKU extension containing the KeyPurposeIds id-kp-serverAuth or anyExtendedKeyUsage, unless the certificate is being issued to the CA itself.</p>
+  <p>Effective for certificates with a notBefore date of July 1, 2020 or later,
+  end entity certificates MUST include an EKU extension containing KeyPurposeId(s)
+  describing the intended usage(s) of the certificate, and the EKU extension MUST NOT
+  contain the KeyPurposeId anyExtendedKeyUsage.</p>
+  <h3 id="53-intermediate-certificates">5.3 Intermediate Certificates</h3>
+  <p>All certificates that are capable of being used to issue new certificates and that directly or transitively chain to a CA certificate included in Mozilla’s root store MUST be operated in accordance with this policy and MUST either be technically constrained or be publicly disclosed and audited.</p>
+  <p>A certificate is deemed as capable of being used to issue new
+  certificates if it contains an <a href="http://tools.ietf.org/html/rfc5280#section-6.1.4">X.509v3 basicConstraints extension</a>
```

Comment:
Suggestion (non-blocking): this is an insecure link, but redirects to a different (secure) domain. Suggest changing this to the destination?
```suggestion
  certificates if it contains an <a href="https://datatracker.ietf.org/doc/html/rfc5280#section-6.1.4">X.509v3 basicConstraints extension</a>
```

Reply:
Good catch. Let's go ahead and make this change here and I'll let them know to make the same fixes in the markdown (or I'll do it and submit a PR). 👍